src/surge-xt/SurgeSynthProcessor.cpp

```diff
@@ -182,6 +196,19 @@ const juce::String SurgeSynthProcessor::getProgramName(int index)
 
 void SurgeSynthProcessor::changeProgramName(int index, const juce::String &newName) {}
 
+/* OSC (Open Sound Control) */
+bool SurgeSynthProcessor::initOSC(int port) {
```

Comment:
The caller isn't doing anything with the return value here; should it be?

Reply:
Added proper error handling/notification.

Reply:
Oops, you're right. Good catch.